app/helpers/numbers_helper.rb

```diff
@@ -2,16 +2,21 @@
 
 module NumbersHelper
   def numbers # rubocop:disable Metrics/AbcSize
-    return @numbers if @numbers.present?
+    Rails.cache.fetch('numbers', expires_in: 24.hours) do
```

Comment:
This PR caches the numbers for 24 hours. @DSRU can we cache them for longer? My assumption is that would almost never change so caching for multiple days seems feasible. We can always manually empty the cache if we needed to push a change through.

Reply:
@eoinkelly yes, they can be cached for longer or even treatd as a static page and I can supply static images in the same way as is done for fingerspelling and classifier pages.

Reply:
I have created issue #576 to track making it a static page. For now I am going to increase the cached time to 7 days  .